src/Gui/NaviCube.cpp

```diff
@@ -204,11 +216,15 @@ class NaviCubeImplementation : public ParameterGrp::ObserverType {
 	QColor m_HiliteColor;
 	QColor m_ButtonColor;
 	QColor m_FrontFaceColor;
+	QColor m_EdgeFaceColor;
```

Comment:
Same here. I think this is not necessary to fix the rendering issue but a feature addition.